packages/flutter_test/lib/src/window.dart

```diff
@@ -518,6 +519,20 @@ class TestWindow implements ui.SingletonFlutterWindow {
   }
 }
 
+/// Convenient methods for auto reset test values
+// Use an `extension`, such that classes implementing TestWindow will not be affected
+extension TestWindowExtensionValueTearDown on TestWindow {
```

Comment:
The Flutter style guide [says](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#tldr) to avoid `extension`.  Is there any way to improve this without using `extension`?

Reply:
By implementing it directly into the TestWindow. This has the slight drawback that anyone who `implements` TestWindow will need to add one more method, but if few people are doing so then looks ok